chromadb/__init__.py

```diff
@@ -165,6 +173,13 @@ def HttpClient(
     if settings is None:
         settings = Settings()
 
+    # Make sure paramaters are the correct types -- users can pass anything.
```

Comment:
should the int conversions have a try/catch in case of failure?

Reply:
I think we're fine erroring if someone passes in a port that's not an int (same applies to other types). All we would do is throw another error.